rapids-4-spark-tools/src/test/scala/com/nvidia/spark/rapids/tool/profiling/ApplicationInfoSuite.scala

```diff
@@ -50,7 +50,9 @@ class ApplicationInfoSuite extends FunSuite with Logging {
       assert(apps.size == 1)
       assert(apps.head.sparkVersion.equals("3.1.1"))
       assert(apps.head.gpuMode.equals(true))
-
+      assert(apps.head.jobStart(apps.head.index).jobID.equals(1))
+      assert(apps.head.stageSubmitted(apps.head.index).numTasks.equals(1))
+      assert(apps.head.taskEnd(apps.head.index).successful.equals(true))
```

Comment:
not much tests added for all these would like to see more

Reply:
Added couple of tests. Will add more tests.

Reply:
lets followup with adding more after this pr